pkg/codegen/docs/templates/function.tmpl

```diff
@@ -0,0 +1,59 @@
+{{ template "header" .Header }}
+
+{{ htmlSafe .Comment }}
+
+{{ .DeprecationMessage }}
+
+<!-- Input properties -->
+## Using {{ .Header.Title }}
+
+{{ htmlSafe "{{< langchoose csharp nojavascript >}}" }}
+
+<!-- TS/JS -->
+<div class="highlight"><pre class="chroma"><code class="language-typescript" data-lang="typescript"><span class="k">function </span>get{{ .ResourceName }}<span class="p">(</span>{{ htmlSafe .FunctionArgs.nodejs }}<span class="p">): Promise<{{ template "linkify_param" .FunctionResult.nodejs }}></span></code></pre></div>
+
+<!-- Python -->
+<div class="highlight"><pre class="chroma"><code class="language-python" data-lang="python"><span class="k">function </span> get_{{ pyName .ResourceName }}(</span>{{ htmlSafe .FunctionArgs.python }}<span class="p">)</span></code></pre></div>
+
+<!-- Go -->
+<div class="highlight"><pre class="chroma"><code class="language-go" data-lang="go"><span class="k">func </span>Lookup{{ .ResourceName }}<span class="p">(</span>{{ htmlSafe .FunctionArgs.go }}<span class="p">) (*{{ template "linkify_param" .FunctionResult.go }}, error)</span></code></pre></div>
+
+<!-- C# -->
+<div class="highlight"><pre class="chroma"><code class="language-csharp" data-lang="csharp"><span class="k">public static </span>Task<{{ template "linkify_param" .FunctionResult.csharp }}> <span class="p">Get{{ .ResourceName }}(</span>{{ htmlSafe .FunctionArgs.csharp }}<span class="p">)</span></code></pre></div>
```

Comment:
This may be changing for 2.0, just so you're aware: https://github.com/pulumi/pulumi-terraform-bridge/issues/125

In which case, we'd probably want to render the static class along with the `InvokeAsync` signature.

```cs
public static class GetVpc
{
    public static Task<GetVpcResult> InvokeAsync(GetVpcArgs? args = null, InvokeOptions? options = null);
}
```

Reply:
Ah thanks for the heads-up. If what is being proposed in the issue is implemented, I could see this being changed to `GetVpc.InvokeAsync(...)` rather than render the static class along with the InvokeAsync signature here. Is there a reason to want to render the class and the static method in its entirety. One reason I could of is that it would be a little more close to what the API doc would show.